domains/sid.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Siddhartha Patki's development portfolio",
+  "repo": "https://github.com/SiddPatki",
+  "owner": {
+    "username": "SiddPatki",
+    "email": "siddharthapatki@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://siddpatki.github.io/my-site/",
```

Comment:
CNAME records cannot be a full url. It has to be a host name. I'd suggest you rename your repo to `siddpatki.github.io`.

You can take a look at the [github pages guide](https://guides.github.com/features/pages/).

Reply:
Sorry and thanks...I am a noob in this so was not sure.
I will update the CNAME as per your comment.
Will I be able to change this CNAME in the future if I want it to redirect to some different link

Reply:
You can change the record any time you want with a new pull request